src/momento/internal/aio/_scs_data_client.py

```diff
@@ -135,7 +143,25 @@ async def execute_delete_request_fn(req: _DeleteRequest) -> _DeleteResponse:
 
     # DICTIONARY COLLECTION METHODS
 
-    # LIST COLLECTION METHODS
```

Comment:
I think we may get merge conflicts unless we keep the line above

Reply:
Whether we think of collections as "not existing" vs "being empty" is something we need to document for the user. I guess we implicitly destroy an empty collection, so "not existing". I think we need a little doc section explaining that.